examples/JoystickControl/JoystickControl.ino

```diff
@@ -0,0 +1,56 @@
+#include 
+Servo servo1;     // 2 Servos Initialized 
+Servo servo2;
+int x_key = A1;     //Horizontal pin for Joystick                                         
```

Comment:
Arduino uses camelCase for our variables. Please spend some time studying the existing example sketches to get an idea of our coding style, then use that coding style in the examples you propose so that we don't end up with an unprofessional inconsistency from one example to another. This sort of inconsistency can cause confusion for beginners.

Reply:
Thank you for being so patient with me, These are my first few contributions. I changed the variable names to be in camelCase. I shall keep these in mind for my next contributions.
I noticed many examples in other libraries do not follow the convention, shall I go ahead and change them?
Thank you